other_types_of_ownership/multi_sig.md

```diff
@@ -152,9 +152,11 @@ Transaction fullySigned =
                     .CombineSignatures(aliceSigned, bobSigned);
 ```
 
-The transaction is now ready to be sent to the network.
+The transaction is now ready to be sent to the network, but notice that the CombineSignatures() method was critical here, because both the aliceSigned and the bobSigned transactions were only partially signed, therefore not acceptable by the network. CombineSignatures() combined the two partially signed transactions into one fully signed transaction.  
+
+> Sidenote: there is an inherent difficulty which arises from this situation. You need to send the newly created, unsigned multi-sig transaction to every signer and after their signed it, you also need to collect the partially signed transactions from them and combine them into one, so that you can publish that on the network. This problem is partially solved by the [BIP-0174](https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki), because it at least standardizes the data format, but you still need to implement your own way to distribute the data between the signing parties.
```

Comment:
I was not aware of BIP 174. It should be noted it is not implemented in NBitcoin right now.